### ISSUE TITLE
[Bug]: Ice Seeds Uncraftable

### ISSUE THREAD

**Issue by sme23** [FTBTeam/FTB-Modpack-Issues#919]
### Modpack

FTB Plexiglass Mountain

### Modpack version

1.0.2

### Describe the bug

The Infusion Crafting recipe for Ice Seeds contains the empty tag `#forge:ice`, leaving them uncraftable.

### Steps to reproduce

1. Click on Ice Seeds in REI.
2. Click the Infusion Crafting tab.
3. See the barrier blocks denoting an empty tag.

### Expected behaviour

Crafting Ice Seeds via Infusion Crafting

### Screenshots

![image](https://user-images.githubusercontent.com/24559482/172516863-b1502c3c-6d21-496b-8620-66a81ea9b9f5.png)

**Comment by Raidobw2** [comment]
If you want it now in tags.js add this then type /reload in-game:
```
    //Ice
    event.add('forge:ice', 'minecraft:ice');
    event.add('forge:ice', 'minecraft:packed_ice');
    event.add('forge:ice', 'minecraft:blue_ice');
    event.add('forge:ice', 'soul_ice:soul_ice');
```
![image](https://user-images.githubusercontent.com/17010632/172539330-3ef58122-0b6c-4e70-8729-0b75a737801f.png)
Going to be fixed in 1.1.0, which is not out yet

**Comment by Raidobw2** [comment]
Fixed in 1.1.0
